src/components/Picker/index.js

```diff
@@ -102,10 +104,28 @@ class Picker extends PureComponent {
         };
 
         this.onInputChange = this.onInputChange.bind(this);
+        this.placeholder = this.props.placeholder;
+        this.items = this.props.items;
     }
 
     componentDidMount() {
         this.setDefaultValue();
+
+        // Windows will reuse the text color of the select for each one of the options
+        // so we might need to color accordingly so it doesn't blend with the background.
+        if (getOperatingSystem() === CONST.OS.WINDOWS) {
+            this.placeholder = {
+                ...this.placeholder,
+                color: '#002140',
+            };
+
+            this.items = _.map(this.items, item => (
+                {
+                    ...item,
+                    color: '#002140',
```

Comment:
We should add this as a CONST

Reply:
Ideally, it should be in the themes but Shawn said we don't have this one yet. DO you want me to add it as constant no matter what?

Reply:
It seems like we have a colors switch PR coming up, based on [this comment](https://github.com/Expensify/App/blob/71e3c907812aee174e62aca5d2a1839e6ebe4d69/src/styles/colors.js#L37). I'll ping Georgia to clean this up once we deprecate this other colors.

Reply:
I would love it if this was moved to a constant or maybe to our `style.variables`. Anytime we leave code like this, even temporarily, that's opening chances for contributors to see it and copy it, assuming it's a good pattern to follow.

From what I've heard, we are not moving quickly on that theme-switcher, and even when they do, it will be hard to find this and fix it, so it could live here for a long time if someone forgets about it.